auth/standard_role_manager.cc

```diff
@@ -651,9 +664,13 @@ future<> standard_role_manager::remove_attribute(std::string_view role_name, std
             meta::role_attributes_table::name);
     if (legacy_mode(_qp)) {
         co_await _qp.execute_internal(query, {sstring(role_name), sstring(attribute_name)}, cql3::query_processor::cache_internal::yes).discard_result();
+        co_return std::vector<mutation>();
     } else {
-        co_await announce_mutations(_qp, _group0_client, query,
-                {sstring(role_name), sstring(attribute_name)}, &_as);
+        if (!timestamp) {
+            on_internal_error(log, "no raft timestamp on v2 auth request");
+        }
+
+        co_return co_await _qp.get_mutations_internal(query, internal_distributed_query_state(), *timestamp, {sstring(role_name), sstring(attribute_name)});
```

Comment:
I'm also not sure about the name `remove_attribute_internal`. I guess you named it like this because we have `get_mutations_internal`? I think the "get" part is actually more important then "internal", i.e. I propose naming this function `get_remove_attribute_mutations`.

Reply:
or even remove_attribute_mutations , get is easy to infer from the context, we are not in java land